### PR TITLE
Update sensor.py

### PR DESCRIPTION
**Pull request by Nivek-domo**
I propose a modification of the sensor.py file in order to add the recognition of PTM 210 or PTM 535 type sensors from enocean. I just tested on my raspberry pi 3 with the latest official Home assistant image https://community.home-assistant.io/t/enocean-rocker-switch-visible-in-log-but-cant-use-entity/492622, and a modification of windows handle

### CODE REVIEW

**Comment by Nivek-domo** [comment]
so in configuration.yaml

`sensor:
  - name: "cara"
    platform: enocean
    id: [0xFE,0xF5,0x60,0x31]
    device_class: windowhandle
  - name: "boutonbur"
    platform: enocean
    id: [0x00,0x32,0x20,0x6F]
    device_class: wallswitch`
image

best regards

**Comment by RomeHein** [comment]
Hello @Nivek-domo !
Thank you very much for the PR.I'm really sorry I took that long to have a look at it 🙌.
The PR LGTM, I would just suggest to split it into two different PRs since we have two different things in the current one :
- the fix for the window handle. Btw could you give more details on this ? Were the payload wrong since the beginning ?
- the new part that handles new device 👏
Thanks in advance !

**Comment by Nivek-domo** [comment]
Hello, I work in a factory that manufactures connected handles, (but not only). We are part of the Enocean alliance, and recently, a client asked us to integrate with HA. This is the reason why I am interested today in this version of home automation.
In some of our handles, we use this "00" error code to report an anomaly, such as a shock or tearing, so that the home automation system understands the abnormal status which could mean a window break-in. This code is tolerated in the alliance but is not officially registered in the EEP. I think it is now a story of a few weeks.
As for the wall switch, I find it a shame not to have it natively on the dashboard, because for me it's a sensor in the same way as a handle... I find it more comfortable to view and use because it is part of the same sensor family. in this way, the state is more easily managed, and the light control with progressive ignition becomes easier to use (when the button is released, a progressive increase in light intensity can be stopped. Pressing it triggers a counter that controls the increment of the brightness of the bulb, and the release stops the counter).
In short, I say this for ease of use.
